### DISCUSSION TITLE
wiki browser on PostgreSQL-based search engine

### DISCUSSION THREAD

**Discussion by hello99974** [requarks/wiki#7358] [Help / Questions] [answered]
Hello, I have little problem with my wiki browser. It searches content for me, but after I click on my search result it doesn't moves me to the place wheres  included. Im forced to use ctrl+F, then it takes me to the place wheres included. 
To be precise: after I click on FINUS as it is showed on the screen below - then it takes me to Finus section but doesnt move me to the place where 'roku' is included. Any hints?
![Zrzut ekranu 2024-09-20 102747](https://github.com/user-attachments/assets/d79a58c4-4ea5-454d-a252-5b8e01d0d21e)

**Comment by NGPixel** [comment, marked as answer]
This is not yet supported but planned for a future release.
